src/v/pandaproxy/rest/configuration.cc

```diff
@@ -54,6 +56,31 @@ configuration::configuration()
       "consumer_instance_timeout_ms",
       "How long to wait for an idle consumer before removing it",
       {},
-      std::chrono::minutes{5}) {}
-
+      std::chrono::minutes{5})
+  , client_cache_max_size(
+      *this,
+      "client_cache_max_size",
+      "The maximum number of kafka clients in the LRU cache",
+      {.needs_restart = config::needs_restart::no},
+      10,
+      [](const size_t& max_size) {
+          std::optional<ss::sstring> msg{std::nullopt};
+          if (max_size <= 0) {
```

Comment:
A question about good practice: What's the preferred method to check for value wrap-around? I could use INT LIMITS but that's system dependent. I could do `max_size > -1` which is a "magic" way to account for this per-system but looks weird since it's a size_t. Or is this simply not our problem at this point?

Reply:
I tried passing `-1`, it's silently ignored, the default is chosen.

Reply:
> A question about good practice: What's the preferred method to check for value wrap-around?

i'm not really sure how it works for config:: stuff. i think there are built-in features for bounds checking. my comment is just that from first principles, by the time this lambda gets around to running any wrap around has already taken place and the inequality >= 0 is always true because size_t is unsigned.

Reply:
> > A question about good practice: What's the preferred method to check for value wrap-around?
> 
> i'm not really sure how it works for config:: stuff. i think there are built-in features for bounds checking. my comment is just that from first principles, by the time this lambda gets around to running any wrap around has already taken place and the inequality >= 0 is always true because size_t is unsigned.

Agreed. I'll remove for now and let built-in features for bounds checking occur.